admin/partials/boldgrid-backup-admin-support.php

```diff
@@ -77,12 +77,12 @@
 		wp_kses(
 			/* translators: 1: HTML anchor for the URL address for BoldGrid Central, 2: HTML anchor close tag.. */
```

Comment:
Update the translators comment like this:
`1: HTML anchor for the URL address for BoldGrid Support ticketing`